src/repositories_services_rust.tf

```diff
@@ -44,6 +45,45 @@ locals {
       }
     }
   }
+
```

Comment:
In my last commit, I moved all vars into a map instead of prefixing them. You can move your `svc_files` map to this `svc` map as well, so we'll have 1 map with `svc` specific configurations.
The repositories prefixed with `lib` can be created in their own file where all configurations can be part of the `lib` local map.